ui/pages/extension/AccessMesheryModal.js

```diff
@@ -0,0 +1,106 @@
+import React, {
+} from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import { Typography, Grid, Button, IconButton } from '@material-ui/core';
+import CloseIcon from '@material-ui/icons/Close';
+
+const styles = makeStyles((theme) => ({
+  paper : {
+    position : 'absolute',
+    width : "30rem",
+    height : "20rem",
+    backgroundColor : theme.palette.background.paper,
+    boxShadow : theme.shadows[5],
+    borderRadius : "4px",
+    top : "0",
+    bottom : "0",
+    left : "0",
+    right : "0",
+    margin : "auto",
```

Comment:
use flex, please.

Reply:
This media query doesn't seem to be having an effect, nor should we need it if you use less fragile CSS.

<img width="576" alt="Screenshot 2022-12-16 at 4 31 45 PM" src="https://user-images.githubusercontent.com/7570704/208199879-76f55e72-d427-4ae4-acf6-82d15fc43779.png">